libraries/AP_Notify/AP_Notify.cpp

```diff
@@ -94,12 +94,17 @@ AP_Notify *AP_Notify::_singleton;
 #endif // BUILD_DEFAULT_LED_TYPE
 
 #ifndef BUZZER_ENABLE_DEFAULT
-#define BUZZER_ENABLE_DEFAULT 1
+#if HAL_ENABLE_LIBUAVCAN_DRIVERS
+// Enable Buzzer messages over UAVCAN
+#define BUZZER_ENABLE_DEFAULT (Notify_Buzz_Builtin | Notify_Buzz_UAVCAN)
+#else
+#define BUZZER_ENABLE_DEFAULT Notify_Buzz_Builtin
+#endif
 #endif
 
 #ifndef BUILD_DEFAULT_BUZZER_TYPE
 #if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS && HAL_DSHOT_ALARM
-  #define BUILD_DEFAULT_BUZZER_TYPE (BUZZER_ENABLE_DEFAULT | (HAL_DSHOT_ALARM << 1))
+  #define BUILD_DEFAULT_BUZZER_TYPE (BUZZER_ENABLE_DEFAULT | Notify_Buzz_DShot)
```

Comment:
I think this enables the dshot alarm by default even if you don't have support for dshot or dshot alarms which I think is wrong.

Reply:
there's a check `#if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS && HAL_DSHOT_ALARM` right above, it should enter this code unless `HAL_DSHOT_ALARM = 1`